### ISSUE TITLE
Missing MetaData files

### ISSUE THREAD

**Issue by hydragosh** [PixelBrownieSoftware/Magnum-Foundation-experimental#2]
When I open it with Unity, Unity console msg:

"Rebuilding library because the metadata file could not be found".

**Comment by PixelBrownieSoftware** [comment]
i've been looking into the issue and it's far more complicated than i thought.
when i opened the scene file demonstrated in the video, there was a lot of missing stuff - since i abandoned this system a while ago, i don't remember a ton about this.
very sorry, i'll try to get it sorted.

in the meantime, copy the scripts from the Globals and Shakespere folder and go from there.

**Comment by hydragosh** [comment]
I think I have put most of the scripts and stuffs back to the right places and make most of the things run accordingly.

But there're a few things I find them problematic, with one being --

After my computer reboot, when I re-opened this Unity Project again, the field of the tr_dialogue is gone.
I have also created my custom TrackAsset scripts and try generating a custom Track in the Timeline. But the results are the same- the empty fields which allow gameObject assignment in the Track scripts, for both tr_dialogue and my custom track script, do not appear.

I also found that at the top of the Hierarchy list, there are a bunch of "Recovering GameObject(xxxxxxxx)" <--xxxxxxx being a 10-digit number. 'not too sure if this is related to the TrackAsset script. 

---------------
I think I know the core idea of how these Timeline scripts work together.
Basically there're 3 key types of script-- the Playable Behavior, the Clip and the Track script.
The Clip script would clone "things" when played, hence producing some unwanted files after a few plays.

**Comment by hydragosh** [comment]
Oh, the timeline will get refreshed after I created another timeline. Now all the empty fields appear!

**Comment by PixelBrownieSoftware** [comment]
i think i remember dealing with the disappearing stuff when i worked on this.

since some have expressed interest in this, i will try to test and fix some of these issues.

although atm i am working on another project, so maintain realistic expectations.
